test/Microsoft.AspNetCore.Routing.Tests/RouteValuesAddressSchemeTest.cs

```diff
@@ -325,6 +369,7 @@ private CustomRouteValuesBasedAddressScheme CreateAddressScheme(params EndpointD
             string template,
             object defaults = null,
             object requiredValues = null,
```

Comment:
Rename to metadataRequiredValues to be clearer about its difference with routePatternRequiredValues